app/components/navigationComponents/BackButtonHeader/styles.js

```diff
@@ -1,8 +1,9 @@
 import { StyleSheet } from 'react-native';
+import { HEADER_ICON_HEIGHT, HEADER_ICON_WIDTH } from '@constants/icons';
 
 export const styles = StyleSheet.create({
   image: {
-    width: 20,
-    height: 20
+    width: HEADER_ICON_WIDTH,
+    height: HEADER_ICON_HEIGHT
```

Comment:
no, i mean that they are the same because is a square style. You should always user the same constant variable in both so the change is reflected in both of them